src/System.Reflection.DispatchProxy/src/System/Reflection/DispatchProxyGenerator.cs

```diff
@@ -477,10 +476,9 @@ private MethodBuilder AddMethodImpl(MethodInfo mi)
                 packedArr.EndSet(typeof(DispatchProxy));
 
                 // packed[PackedArgs.DeclaringTypePosition] = typeof(iface);
-                MethodInfo Type_GetTypeFromHandle = typeof(Type).GetRuntimeMethod("GetTypeFromHandle", new Type[] { typeof(RuntimeTypeHandle) });
+                MethodInfo Type_GetTypeFromHandle = typeof(Type).GetRuntimeMethod("GetTypeFromHandle", new Type[] { typeof(RuntimeTypeHandle) })!;
                 int methodToken;
-                Type declaringType;
-                _assembly.GetTokenForMethod(mi, out declaringType, out methodToken);
+                _assembly.GetTokenForMethod(mi, out Type declaringType, out methodToken);
```

Comment:
Nit: methodToken can also be declared inline.

Reply:
true